duckduckgo/iOS
# add atb integration tests

Task/Issue URL: https://app.asana.com/0/414235014887631/724275184893418
Tech Design URL:
CC:

**Description**:

Adds integration tests for ATB.  

Note, some changes were required to AppUrls - comments are inline.

**Steps to test this PR**:

1. Check out the code and execute `./run_atb_tests.sh`
1. Observe that Travis runs the build successfully
1. Regression test the app

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
